olive/passes/onnx/qnn/qnn_mixed_precision_overrides.py

```diff
@@ -0,0 +1,216 @@
+                        and not handle_conflict(node.input[1], node)
+                    ):
+                        add_initializer_tensor_to_16bit_overrides(node.input[1])
+
+                elif (
+                    node.input[1] in activations_16bit
+                    and node.input[0] not in activations_16bit
+                    and node.input[0] in init_map
+                    and not handle_conflict(node.input[0], node)
+                ):
+                    add_initializer_tensor_to_16bit_overrides(node.input[0])
+
+        model_modified = False
+        for tensor_name, tensor_attrs in conflict_data.items():
+            if tensor_attrs["counts"] == 0:
+                # If initializer tensor is input to multiple nodes but all of them have 16bit constraint
+                # convert the tensor regularly
+                add_initializer_tensor_to_16bit_overrides(tensor_name)
+            else:
+                # If not then we create copy of the initializer that will be converted to 16 bits while
+                # original one will be quantized to 8 bits
+                copy_of_initializer = onnx.TensorProto()
+                copy_of_initializer.CopyFrom(init_map[tensor_name])
+                copy_of_initializer.name = copy_of_initializer.name + "_MixedPrecision_copy"
+                onnx_model.graph.initializer.extend([copy_of_initializer])
+
+                for node in tensor_attrs["nodes"]:
+                    for i, item in enumerate(node.input):
+                        # for all nodes that require their input to be 16 bit, we replace input tensor with copy
+                        node.input[i] = copy_of_initializer.name if item == tensor_name else item
+                # and assign convert operator to it
+                add_initializer_tensor_to_16bit_overrides(copy_of_initializer.name)
+                model_modified = True
+
+        if not model_modified:
+            logger.info("Model %s does not have any mixed precision conflicts", model.model_path)
+            return model
+
+        onnx_model = ONNXModel(onnx_model)
+        onnx_model.topological_sort()
+        output_model_path = resolve_onnx_path(output_model_path, Path(model.model_path).name)
+        model_proto_to_file(
+            onnx_model.model,
+            output_model_path,
+            save_as_external_data=config["save_as_external_data"],
+            all_tensors_to_one_file=config["all_tensors_to_one_file"],
+            external_data_name=config["external_data_name"],
+            size_threshold=config["size_threshold"],
+            convert_attribute=config["convert_attribute"],
+        )
+
+        overrides_jsonable = {
+            tensor: [{"quant_type": quant["quant_type"].name} for quant in quant_list]
+            for tensor, quant_list in overrides.items()
+        }
+        return ONNXModelHandler(output_model_path, model_attributes={"mixed_precision_overrides": overrides_jsonable})
```

Comment:
Also include the attributes from the input_model here? Since this handler has attributes, the pass won't automatically carry over the input model's attributes